domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/response/Response.java

```diff
@@ -33,6 +34,8 @@
 
     private final YesNoOption freeMediation;
```

Comment:
Will you need this after encapsulating this inside `FreeMediation` object.

Reply:
doesn't apply after latest changes